app/controllers/concerns/api/admin/users_controller.rb

```diff
@@ -5,7 +5,7 @@ module UsersController
 
       def create
         # NOTE: We can add an inviting user here, e.g. User.invite!(current_user, user_params).
-        User.invite!(user_params)
+        User.invite!(user_params.merge(registered: false))
```

Comment:
Probably minor, but [it _seems like_](https://github.com/forem/forem/commit/b1e7d8a5ec17eb62c24b2b5d935e4841e59fa0ec#diff-9802ca3c9c4cf89904fd44bc114e35ebdf2c5dd3d5b645491e2b253e1afef29b) we expect "invited but not yet answered the invite" to be signaled by `registered: false` — which _probably_ means the `registered: false` attribute management belongs in the model's `invite!` method... but, since we get the `invite!` method by way of `devise_invitable`, that would mean we either need a local override + `super` in `User.invite!` _or_ use the `before_invitation_created` callback [provided by the gem](https://github.com/scambra/devise_invitable#callbacks-). (If we'd handled it in the model [originally](https://github.com/forem/forem/blob/da6a6739e56df3857dcb2d8dc5f12935fd0d3823/app/controllers/internal/invitations_controller.rb#L15-L19), we wouldn't have to patch it in each controller instance.)